lib/galaxy/datatypes/blast.py

```diff
@@ -0,0 +1,281 @@
+"""NCBI BLAST datatypes.
+
+Covers the ``blastxml`` format and the BLAST databases.
+"""
+
+import logging
+import os
+from time import sleep
+
+from galaxy.datatypes.data import get_file_peek
+from galaxy.datatypes.data import Data, Text
+from galaxy.datatypes.xml import GenericXml
+
+log = logging.getLogger(__name__)
+
+
+class BlastXml(GenericXml):
+    """NCBI Blast XML Output data"""
```

Comment:
> But is it http://edamontology.org/data_1381 Alignment of exactly two molecular sequences? In general a blastxml file contains multiple pairwise alignments from many query sequences and against multiple matched sequences. The parent entry http://edamontology.org/data_0863 Alignment of multiple molecular sequences sounds safer to me.

I thought, the alignments in a blastxml file are exactly of two sequences :)

> Can you have multiple EDAM tags? http://edamontology.org/format_3331 XML format as produced by the NCBI Blast package seems like the best choice if we can only pick one.


But is it http://edamontology.org/data_1381 Alignment of exactly two molecular sequences? In general a blastxml file contains multiple pairwise alignments from many query sequences and against multiple matched sequences. The parent entry http://edamontology.org/data_0863 Alignment of multiple molecular sequences sounds safer to me.

Can you have multiple EDAM tags? http://edamontology.org/format_3331 XML format as produced by the NCBI Blast package seems like the best choice if we can only pick one.

I think we can and should have only one. Go for the saver one if @odoppelt does not reply. She is our edam expert. Maybe Herve (@hmenager) can help as well?